### PR TITLE
docs(README): add JSON examples

### PR DESCRIPTION
**Pull request by JanWennrich**
To use YAML config files the "symfony/yaml" package is required. This hinders a "quick" start.

JSON config files can be used out of the box.
Thus examples for JSON config files are added.

### CODE REVIEW

**Comment by JanWennrich** [comment]
This PR is missing the PHPStan fix from the other PR.  
Let me rebase it real quick

**Comment by Crell** [comment]
I really like the idea of using `details` to show both examples!  When I view the page in GitHub, though, it's showing them both collapsed.  Is there a way to force one to be open by default?

**Comment by JanWennrich** [comment]
> I really like the idea of using `details` to show both examples! When I view the page in GitHub, though, it's showing them both collapsed. Is there a way to force one to be open by default?

There is the `open` attribute for the `<details>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#open  
It seems like the GitHub markdown parser supports it:

<details open>
<summary>Open details element</summary>
Lorem Ipsum
</details>

<details>
<summary>Default details element</summary>
Foo Bar
</details>

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Because each config object is its own class, it integrates seamlessly with a Dep
 
 Let's see an example.
 
+<details>
```
Comment (Crell) [inline README.md]:
```suggestion
<details open>
```

File: README.md [hunk 2]
```diff
@@ -107,6 +157,11 @@ Note: Make certain the directory where the file cache is stored is not publicly
 
 Example:
 
+<details>
```
Comment (Crell) [inline README.md]:
```suggestion
<details open>
```

**Comment by Crell** [comment]
Perfect.  I went ahead and added those.  Thanks.

**Pull request merged**